sources/packages/backend/workflow/test/test-utils/mock/assessment-gateway/load-assessment-consolidated-data-mock.ts

```diff
@@ -1,19 +1,23 @@
-import { WorkflowServiceTasks } from "../..";
-import { createMockedWorkerResult } from "..";
+import { WorkflowServiceTasks, WorkflowSubprocesses } from "../..";
+import { WorkerMockedData } from "..";
 import { AssessmentConsolidatedData } from "../../../models";
 
 /**
- * Creates the mock for the 'Load consolidated data' completed subprocess.
+ * Creates the mock for the 'Load assessment data task' completed.
  * @param assessmentConsolidatedData assessment consolidated data.
- * @returns mock for the 'Load consolidated data' completed subprocess.
+ * @returns mock for the 'Load assessment data task' completed.
  */
-export function createLoadAssessmentConsolidatedDataMock(options: {
+export function createLoadAssessmentDataTaskMock(options: {
   assessmentConsolidatedData: AssessmentConsolidatedData;
-}): Record<string, unknown> {
-  return createMockedWorkerResult(
-    WorkflowServiceTasks.LoadAssessmentConsolidatedData,
-    {
+  subprocess:
```

Comment:
pls add comments and review the commnets

Reply:
Comments updated.